### PR TITLE
Compact list

### PR DESCRIPTION
**Pull request by jimmeak**

### CODE REVIEW

**Comment by Almad** [comment]
`Skupiny` are not working in some cases (like Articles or Warrior)

image

I'd also maybe add more creations per page in those cases 🤔

**Comment by Almad** [comment]
I think the singular idea doesn't make sense in this context, the correct name for the column is just "Name". 

Or do you think the "Válečník" column above makes sense? IMHO it only goes for "Článek" in ČaE, somewhat.

File: ddcz/templatetags/pages.py [hunk 1]
```diff
@@ -7,3 +7,9 @@
 @register.inclusion_tag("creative-pages/pagination-links.html")
 def pagination(articles, per_page=False):
     return {"articles": articles, "per_page": per_page}
+
+
+@register.inclusion_tag("creative-pages/list-default.html")
+def list_default(articles, page_slug=False, skin="light"):
+    print("\n\n\n\n")
```
Comment (Almad) [inline ddcz/templatetags/pages.py:14]: Leftover?

File: ddcz/templatetags/creations.py [hunk 2]
```diff
@@ -47,6 +48,11 @@ def creation_canonical_url(page, creation):
     return page.get_creation_canonical_url(creation)
 
 
+@register.filter
+def singular_name_from_slug(slug):
+    return CreativePage.objects.get(slug=slug).singular_name
```
Comment (Almad) [inline ddcz/templatetags/creations.py:53]: This is unnecessary additional query since you can pass a creative page from view where you already have it

**Review by Almad — CHANGES_REQUESTED** [review]
I just wonder—instead of doing template tag, wouldn't it be better to remove all `-list` templates and have one generic `creations-list` one? I don't think there is now something creation-specific left in all except (photo)gallery/download anyway, or is it?

**Comment by jimmeak** [comment]
> I think the singular idea doesn't make sense in this context, the correct name for the column is just "Name".
> 
> Or do you think the "Válečník" column above makes sense? IMHO it only goes for "Článek" in ČaE, somewhat.

@Almad Of course, just like this it is not good, but I had the name before and e. g. `Hřbitov` is not very good either. That's why I wanted to make something more suitable exactly for this heading.

File: ddcz/templatetags/pages.py (hunk 1 above)
Comment (jimmeak) [reply ddcz/templatetags/pages.py:14]: yup, thanks :)

**Comment by jimmeak** [comment]
> I just wonder—instead of doing template tag, wouldn't it be better to remove all `-list` templates and have one generic `creations-list` one? I don't think there is now something creation-specific left in all except (photo)gallery/download anyway, or is it?

I believe so, but I didn't want to destroy it all before asking first. :)

**Comment by Almad** [comment]
@jimmeak 

> I believe so, but I didn't want to destroy it all before asking first. :)

Nuke it from orbit! :) 

> Of course, just like this it is not good, but I had the name before and e. g. Hřbitov is not very good either. That's why I wanted to make something more suitable exactly for this heading.

How about "Jméno"? It also removes the need for singular, which I think doesn't make sense on the CreativePage model anyway—it should be on the particular Creations model.

**Comment by jimmeak** [comment]
> How about "Jméno"? It also removes the need for singular, which I think doesn't make sense on the CreativePage model anyway—it should be on the particular Creations model.

I went for a "Název", it seems better for creations than name.

**Comment by jimmeak** [comment]
> Nuke it from orbit! :)

Done, but it still will need some gallery and those special lists to edit, I think we have another issues for it (I believe for @Wynnka :smile: )

**Comment by Almad** [comment]
To be rebased & sent to master

**Pull request closed without merging**